polish import doc (#20019)
pingcap/docs: tidb-cloud/tidb-cloud-import-local-files.md

    2. Click the name of your target cluster to go to its overview page, and then click **Import** in the left navigation pane.

2. On the **Import** page, you can directly drag and drop your local file to the upload area, or click **Upload a local file** to select and upload the target local file. Note that you can upload only one CSV file of less than 250 MiB for one task. If your local file is larger than 250 MiB, see [How to import a local file larger than 250 MiB?](#how-to-import-a-local-file-larger-than-250-mib).

> 3. In the **Destination** section, select the target database and the target table, or enter a name directly to create a new database or a new table. The name must contain characters in Unicode BMP, excluding \u0000 and whitespace characters, and at most contain 64 characters. Click **Define Table**, the **Table Definition** section is displayed.

Reviewer: ```suggestion
3. In the **Destination** section, select the target database and the target table, or enter a name directly to create a new database or a new table. The name must contain characters in Unicode BMP (Basic Multilingual Plane), excluding the null character `\u0000` and whitespace characters, and can be up to 64 characters in length. Click **Define Table**, the **Table Definition** section is displayed.
```

> 3. In the **Destination** section, select the target database and the target table, or enter a name directly to create a new database or a new table. The name must contain characters in Unicode BMP, excluding \u0000 and whitespace characters, and at most contain 64 characters.. Click **Define Table**, the **Table Definition** section is displayed.

Reviewer 2: ```suggestion
3. In the **Destination** section, select the target database and the target table, or enter a name directly to create a new database or a new table. The name must contain characters in Unicode BMP, excluding \u0000 and whitespace characters, and at most contain 64 characters. Click **Define Table**, the **Table Definition** section is displayed.
```

> 3. In the **Destination** section, select the target database and the target table, or enter a name directly to create a new database or a new table. The name must contain characters in Unicode BMP (Basic Multilingual Plane), excluding the null character `\u0000` and whitespace characters, and can be up to 64 characters in length. Click **Define Table**, the **Table Definition** section is displayed.

Reviewer 3: ```suggestion
3. In the **Destination** section, select the target database and the target table, or enter a name directly to create a new database or a new table. The name must only contain characters in Unicode BMP (Basic Multilingual Plane), excluding the null character `\u0000` and whitespace characters, and can be up to 64 characters in length. Click **Define Table**, the **Table Definition** section is displayed.
```

4. Check the table.

    You can see a list of configurable table columns. Each line shows the table column name inferred by TiDB Cloud, the table column type inferred, and the previewed data from the CSV file.

    - If you import data into an existing table in TiDB Cloud, the column list is extracted from the table definition, and the previewed data is mapped to the corresponding columns by column names.

>     - If you want to create a new table, the column list is extracted from the CSV file, and the column type is inferred by TiDB Cloud. For example, if the previewed data is all integers, the inferred column type will be integer.

Reviewer 3: ```suggestion
    - If you want to create a new table, the column list is extracted from the CSV file, and the column type is inferred by TiDB Cloud. For example, if the previewed data is all integers, the inferred column type will be `INTEGER`.
```

    - If you need TiDB Cloud to create the target table, input the name for each column. The column name must meet the following requirements:

>         * The name must be composed of characters in Unicode BMP, excluding \u0000 and whitespace characters

Reviewer: ```suggestion
        * The name must be composed of characters in Unicode BMP, excluding the null character `\u0000` and whitespace characters.
```

### Why can't I query a column with a reserved keyword after importing data into TiDB Cloud?

If a column name is a reserved [keyword](/keywords.md) in TiDB, when you query the column, you need to add backticks `` ` `` to enclose the column name. For example, if the column name is `order`, you need to query the column with `` `order` ``.

### How to import a local file larger than 250 MiB?

If the file is larger than 250 MiB, you can use [TiDB Cloud CLI](/tidb-cloud/get-started-with-cli.md) to import the file. For more information, see [`ticloud serverless import start`](/tidb-cloud/ticloud-import-start.md).

> Or you can use the `split [-l ${line_count}]` utility to split it into multiple smaller files (for Linux or macOS only). For example, run `split -l 100000 tidb-01.csv small_files` to split a file named `tidb-01.csv` by line length `100000`, and the split files are named `small_files${suffix}`. Then, you can import these smaller files to TiDB Cloud one by one.

Reviewer: ```suggestion
Alternatively, you can use the `split [-l ${line_count}]` utility to split it into multiple smaller files (for Linux or macOS only). For example, run `split -l 100000 tidb-01.csv small_files` to split a file named `tidb-01.csv` by line length `100000`, and the split files are named `small_files${suffix}`. Then, you can import these smaller files to TiDB Cloud one by one.
```
